libraries/tokens/CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.3] - 2020-12-09
+
+### Changed
```

Comment:
I would say this should be categorised as a "Removed" according to [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

Reply:
True that.